src/plugin/plugin.rs

```diff
@@ -2,8 +2,7 @@ use crate::pipeline::{CollisionEvent, ContactForceEvent};
 use crate::plugin::configuration::SimulationToRenderTime;
 use crate::plugin::{systems, RapierConfiguration, RapierContext};
 use crate::prelude::*;
-use bevy::ecs::event::Events;
-use bevy::ecs::system::SystemParamItem;
+use bevy::ecs::{event::Events, schedule_v3::SystemConfigs, system::SystemParam};
```

Comment:
It should be using `schedule::SystemConfigs`. `schedule_v3` got renamed to `schedule` in https://github.com/bevyengine/bevy/commit/aa4170d9a471c6f6a4f3bea4e41ed2c39de98e16